sdk/core/azure-core-experimental/azure/core/experimental/transport/_httpx.py

```diff
@@ -124,6 +154,16 @@ def __exit__(self, *args) -> None:
         self.close()
 
     def send(self, request: HttpRequest, **kwargs) -> HttpXTransportResponse:
+        """Send a request and get back a response.
+
+        :param request: The request object to be sent.
+        :type request: ~azure.core.rest.HttpRequest
+        :keyword bool stream: Whether to stream the response. Defaults to False.
+        :keyword int connection_timeout: The timeout setting for the connection.
+        :keyword bool connection_verify: Whether SSL verification is done or not.
```

Comment:
I see you have

kwargs.pop("connection_verify", None)

Seems to me you don't honor it?